StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
+        {
+            setCountryCode(countryCode);
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"All cookies are set")]
+        public void GivenAllCookiesAreSet()
+        {
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR");
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR_ANALYTICS");
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR_SOCIALMEDIA");
+            Driver.Manage().Cookies.DeleteCookieNamed("newsletterSeen");
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR_ANALYTICS", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR_SOCIALMEDIA", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("newsletterSeen", "true"));
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"I have one item in my shopping basket")]
+        public void GivenIHaveOneItemInMyShoppingBasket()
+        {
+            string ShoppingBasket1 = baseURLCK + "BasketRePopulate?ean=8719113837794&qty=1";
+            Driver.Navigate().GoToUrl(ShoppingBasket1);
+        }
+
+        [Given(@"I proceed to the checkout page")]
+        public void GivenIProceedToTheCheckoutPage()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("stickyPanel")).Displayed);
```

Comment:
it's not a good practice to use `Wait.Until`, I guess you are using `Selenium`, so it's not correct usage of selenium.

* When you do `Driver.FindElement`,  it already means you are trying to ensure the element is visible,  if you get the element , then it means the element already there. 
* If you prefer to ensure it's displayed, you could do assertion to check the element attribute after you get the element.